package.json

```diff
@@ -4,13 +4,16 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "version": "pnpm changeset version && pnpm i --no-frozen-lockfile"
+    "version": "pnpm changeset version && pnpm i --no-frozen-lockfile",
+    "format": "prettier -w --cache --plugin prettier-plugin-astro ."
```

Comment:
Make sure this actually formats `.astro` files because there's a bug in Prettier 3 where it doesn't take into account file types added by plugins when just doing `prettier -w .` and you need to manually specify something like `prettier -w **/*.astro`.

Reply:
I _think_ by adding `--plugin prettier-plugin-astro` to the script this takes care of it. I got it to format all of the astro files here by only running `pnpm format` at least.